docker-compose.yml

```diff
@@ -2,7 +2,7 @@
 version: '3.3'
 services:
   elasticsearch:
-    image: elasticsearch:6.4.0
+    image: elasticsearch:7.6.0
```

Comment:
@akmiller01 

For this and `requirements.txt`, I'd advise against bumping ES to something that isn't supported by the Dokku plugin, as the backends need to match up:
- https://docs.wagtail.io/en/v2.8.1/topics/search/backends.html#elasticsearch-backend

Latest ES supported by the Dokku plugin as far as I can tell is `6.8.5`:
- https://github.com/dokku/dokku-elasticsearch#dokku-elasticsearch--

The `requirements.txt`, `Dokku` plugin and `docker-compose` versions are currently all set to `6.4.0` as I think that this was the most compatible at the time of the last upgrade. It may be that they could be upgraded to `6.8.5`, but be aware that to do so requires the unlinking, destroying, creating and relinking of each `Dokku` ES container (which makes search fail on the linked application during that time).

Here's some issues that relate to ES v7:
- https://github.com/dokku/dokku-elasticsearch/issues/72
- https://www.gitmemory.com/issue/dokku/dokku-elasticsearch/72/516100552